Cython/Compiler/TypeStubGenerator.py

```diff
@@ -0,0 +1,433 @@
+from . import Version
+from .Nodes import CNameDeclaratorNode
+from .ExprNodes import CallNode, NameNode, ImportNode, TupleNode, AttributeNode
+from ..CodeWriter import DeclarationWriter
+from .Visitor import CythonTransform
+from . import PyrexTypes
+from ..Utils import open_new_file
+import cython 
+import os 
+import sys 
+
+cython.declare(PyrexTypes=object, Naming=object, ExprNodes=object, Nodes=object,
+               Options=object, UtilNodes=object, LetNode=object,
+               LetRefNode=object, TreeFragment=object, EncodedString=object,
+               error=object, warning=object, copy=object, _unicode=object)
```

Comment:
Note that `error` and `warning` aren't even imported, so declaring them as object probably isn't helpful (but again, I don't think you need this line)